### PR TITLE
Dependency updates: Bump TypeScript and Jest packages

### PR DESCRIPTION
**Pull request by eggplantzzz**

### CODE REVIEW

File: packages/abi-utils/package.json [hunk 1]
```diff
@@ -2,51 +2,51 @@
...
     "prepare": "yarn build",
     "test": "jest lib/**"
   },
-  "bugs": {
-    "url": "https://github.com/trufflesuite/truffle/issues"
+  "types": "dist/lib/index.d.ts",
+  "dependencies": {
+    "change-case": "3.0.2",
+    "faker": "5.5.3",
+    "fast-check": "^2.12.1"
```
Comment (cds-amal) [inline packages/abi-utils/package.json:29]: I realize this isn't part of you PR but wonder why these aren't dev deps.

File: packages/db-kit/package.json [hunk 2]
```diff
@@ -74,5 +52,28 @@
     "meow": "^9.0.0",
     "react": "^16.8.0",
     "source-map-support": "^0.5.19"
```
Comment (cds-amal) [inline packages/db-kit/package.json:54]: Should this be a dev dep?

**Review by cds-amal — COMMENTED** [review]
The code change themselves look good but as a matter of git readability there are two commits that have a `subject: "y"` which devalues `git log`'s value and affected my ability to easily get an overview of the PR using normal git tooling. Could you update those commit messages before merging?

File: packages/abi-utils/package.json (hunk 1 above)
Comment (cds-amal) [reply packages/abi-utils/package.json:29]: Lernaaaaaaaaaaaa

**Review by cds-amal — APPROVED** [review]
Thanks @eggplantzzz ! I like it

**Review by xhulz — APPROVED** [review]

**Pull request merged**